Gemfile.base

```diff
@@ -63,3 +63,4 @@ gem 'bugsnag', '~> 6', require: nil
 gem 'prometheus-client'
 gem 'yabeda-prometheus', '= 0.1.4'
 gem 'async-redis', '~> 0.4.1'
+gem 'falcon', '~> 0.34'
```

Comment:
License of Falcon as a new dependency is OK?
Passes our license_checker?

Reply:
Yes, it's MIT.